LogMonitor/src/LogMonitor/LogWriter.h

```diff
@@ -66,14 +64,7 @@ public :
     {
         AcquireSRWLockExclusive(&m_stdoutLock);
 
-        std::wstring output = LogMessage + L"\n";
-
-        if (m_isConsole) {
-            WriteConsoleW(hConsole, output.c_str(), wcslen(output.c_str()), NULL, NULL);
-        } else {
-            wprintf(output.c_str());
-        }
-
+        wprintf(L"%s\n", LogMessage.c_str());

```

Comment:
This function is a duplicate of #L73

Reply:
They are overloaded functions but exactly the same in how they work. Will explore further outside of this revert
```c++
void WriteConsoleLog(_In_ const std::wstring&& LogMessage) {}
void WriteConsoleLog(_In_ const std::wstring& LogMessage) {}
```